docs/en/sql-reference/sql-statements/data-definition/ALTER_MATERIALIZED_VIEW.md

```diff
@@ -28,9 +28,8 @@ This SQL statement can：
   - All session variable-related properties. For information on session variables, see [System variables](../../../reference/System_variable.md).
 
 :::tip
-
-This operation requires the ALTER privilege on the target materialized view. You can follow the instructions in [GRANT](../account-management/GRANT.md) to grant this privilege.
-
+- This operation requires the ALTER privilege on the target materialized view. You can follow the instructions in [GRANT](../account-management/GRANT.md) to grant this privilege.
+- ALTER MATERIALIZED VIEW does not support the direct alteration of the query statement used to build the materialized view. You can build a new materialized view and swap it with the original one using ALTER MATERIALIZED VIEW SWAP WITH.
```

Comment:
ALTER MATERIALIZED VIEW does not support directly modifying the query statement used to xxx